web/src/composebox_typeahead.ts

```diff
@@ -1321,8 +1329,23 @@ export function initialize_compose_typeahead($element: JQuery<HTMLTextAreaElemen
             updater: content_typeahead_selected,
             stopAdvance: true, // Do not advance to the next field on a Tab or Enter
             automated: compose_automated_selection,
+            option_label(_matching_items, item): string | false {
+                if (item.type === "topic_list" && item.is_stream_only) {
+                    return `<em>${$t({defaultMessage: "(link to channel)"})}</em>`;
+                }
+                return false;
+            },
             trigger_selection: compose_trigger_selection,
             header_html: get_header_html,
+            hideAfterSelect() {
+                // After selecting a stream, we immediately show topic options, so we don't want to hide the typeahead.
+                return completing !== "stream";
+            },
+            getCustomItemClassname(item) {
+                return item.type === "topic_list" && !item.is_stream_only
+                    ? "topic-typeahead-link"
+                    : "";
+            },
```

Comment:
Please write a comment explaining what this is doing. I'm not entirely sure why you need a bootstrap change vs. code in `typeahead_list_item.hbs` in the first place.

Reply:
OK, but how is this different from the existing `option_label` option?

Reply:
From my understanding, `option_label` is html to label a typeahead item, which is different. The change @apoorvapendse is making is adding a custom classname to this typeahead item to style it differently, and it's only sometimes added depending on the item type.